app/react/Templates/components/FormConfigSelect.js

```diff
@@ -100,4 +107,5 @@ export function mapStateToProps(state, props) {
   };
 }
 
+export { FormConfigSelect };
```

Comment:
I don't think this is ever used, can we remove this export?

Reply:
Totally, thank you!